templates/default/voipaccount/voipaccountbillinglist.html

```diff
@@ -318,13 +322,19 @@ <h1>{$layout.pagetitle}</h1>
 </table>
 
 <script>
-
-	$(function() {
-		$('#choosefilter').find('input,select').change(function() {
-			$('#choosefilter').submit();
+	function CustomerChanged(str) {
+		$.ajax({
+			url: '?m=voipaccountbillinglist',
+			type: "POST",
+			data: { str : str },
+			success:
+				function(response) {
+					var selectbox = $('#customernumberselect');
+					var parsed = jQuery.parseJSON( response );
```

Comment:
Czego tu oczkujesz w formacie JSON? Od razu przy $.ajax() możesz poinformować jQuery, że oczekujesz odpowiedzi JSON i otrzymasz od razu sparsowaną. Zobacz w innych miejscach jak to robione.

Reply:
Faktycznie kod wygląda ładniej.

https://github.com/lmsgit/lms/pull/1819/commits/22c02c5c18acaec7b7a1a867eb9b27e5a5a1a645